source/elements/dpcpp/source/index.rst

```diff
@@ -49,151 +48,99 @@ some additional extensions being required, and some DPC++ extensions no longer
-   `Pinned memory property`_    DPC++ extension           Required              Required              Required              NA [#na]_
-   `Reductions`_                DPC++ extension           Required [#redc]_     Required [#redc]_     Not required [#tmp]_  NA [#na]_
+   `Extended atomics`_          DPC++ extension           Required              Required              Required              NA [#na]_
+   `Filter selector`_           DPC++ extension           Required              Required              Required              NA [#na]_
+   `FPGA LSU controls`_         DPC++ extension           Not required          Not required          Required              NA [#na]_
+   `FPGA memory channel`_       DPC++ extension           Not required          Not required          Required              NA [#na]_
+   `FPGA register`_             DPC++ extension           Not required          Not required          Required              NA [#na]_
+   `FPGA selector`_             DPC++ extension           Required              Required              Required              NA [#na]_
+   `GPU device info`_           DPC++ extension           Required              Required              Required              NA [#na]_
+   `Level zero backend`_        DPC++ extension           Not required          Not required          Not required          NA [#na]_
+   `Local memory allocations`_  DPC++ extension           Required              Required              Required              NA [#na]_
+   `Pinned memory property`_             DPC++ extension           Required              Required              Required              NA [#na]_
+   `Platform context`_          DPC++ extension           Required              Required              Required              NA [#na]_
    `Restrict all arguments`_    DPC++ extension           Required              Required              Required              NA [#na]_
-   `Static local mem query`_    DPC++ extension           Not required [#tmp]_  Not required [#tmp]_  Not required [#tmp]_  NA [#na]_
-   `Sub-groups`_                DPC++ extension           Required              Required              Not required          `sub_group <https://github.com/intel/llvm/tree/sycl/sycl/test/sub_group>`__
-   `Sub-group algorithms`_      DPC++ extension           Required [#sga]_      Required [#sga]_      Not required          `sub_group <https://github.com/intel/llvm/tree/sycl/sycl/test/sub_group>`__
+   `Sub-group mask`_            DPC++ extension           Required              Required              Not required          NA [#na]_
    ===========================  ========================  ====================  ====================  ====================  =============
 
 
 ..   ==========================  ================  ================  ====================  =============
 
 .. _`Accessor properties`: https://github.com/intel/llvm/tree/sycl/sycl/doc/extensions/accessor_properties
```

Comment:
I don't think I understand this one.  We are pointing to the spec document here, aren't we?

Reply:
No, you are currently pointing to the directory that contains the spec document.  Actually, you do this for a number of the entries.  It would probably be better to update them all to point directly to the spec document.

Reply:
Got it.  Fixed.